webant/webant.py

```diff
@@ -213,7 +214,7 @@ def download_book(bookid, fileid):
                 except:
                     app.logger.warn("Cannot increment download count",
                                     exc_info=1)
-                return send_file(app.fsdb.getFilePath(file['fsdb_id']),
+                return send_file(app.fsdb.get_file_path(file['fsdb_id']),
```

Comment:
Same here having the get in front of it is soo java :P s/get_file_path/file_path/. The antani rule is valid also here.